src/c++/perf_analyzer/client_backend/triton/triton_client_backend.cc

```diff
@@ -625,6 +641,54 @@ TritonInferResult::RawData(
   return Error::Success;
 }
 
+struct triton::client::HttpSslOptions
+ParseHttpSslOptions(
+    const struct triton::perfanalyzer::clientbackend::SslOptionsBase&
+        ssl_options)
```

Comment:
remove `struct`

Reply:
Also probably can move this to an anonymous namespace (maybe to the top of this file)

Reply:
good catch--removed here and everywhere else applicable 👍